etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-oscapacity.yml

```diff
@@ -1,3 +1,4 @@
+{% if stackhpc_enable_os_capacity | bool %}
```

Comment:
Should this still end with .j2 to keep tox happy?

Reply:
kolla-ansible only reads "*.yml": https://opendev.org/openstack/kolla-ansible/src/branch/master/ansible/roles/prometheus/tasks/config.yml#L58-L66